tests/features/kill_primary.feature

```diff
@@ -344,7 +344,12 @@ Feature: Destroy primary in various scenarios
             state: streaming
         """
         When we stop container "postgresql3"
-        When we drop replication slot "pgconsul_postgresql3_1_pgconsul_pgconsul_net" in container "postgresql1"
+        And we wait "10.0" seconds
```

Comment:
It would be a flapping test, since `drop_slot_countdown` is 10 too.
10 seconds is too soon to guarantee that 10 iterations have passed and slot was dropped.

Reply:
There is retries in ```Then container "postgresql1" has following replication slots``` check

Reply:
There are retries in ```Then container "..." has following replication slots``` check

Reply:
I think it makes the test more explicit. So, I want to leave it as it is.